diff --git a/projects/DensePose/setup.py b/projects/DensePose/setup.py
index a46f47a..116e1c9 100644
--- a/projects/DensePose/setup.py
+++ b/projects/DensePose/setup.py
@@ -38,7 +38,7 @@ setup(
     install_requires=[
         "av>=8.0.3",
         "detectron2@git+https://github.com/facebookresearch/detectron2.git",
-        "opencv-python-headless>=4.5.3.56",
+        "opencv-contrib-python>=4.5.3.56",
         "scipy>=1.5.4",
     ],
 )
