diff --git a/setup.py b/setup.py
index 2cf5d69..53fb421 100644
--- a/setup.py
+++ b/setup.py
@@ -183,7 +183,7 @@ setup(
         # NOTE: when updating fvcore/iopath version, make sure fvcore depends
         # on compatible version of iopath.
         "fvcore>=0.1.5,<0.1.6",  # required like this to make it pip installable
-        "iopath>=0.1.7,<0.1.10",
+        "iopath>=0.1.7,<0.1.11",
         "dataclasses; python_version<'3.7'",
         "omegaconf>=2.1,<2.4",
         "hydra-core>=1.1",
